README.md

```diff
@@ -14,4 +14,4 @@ The `App`contains a `MoviesList` and a search field. Implement filtering using a
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_movies-list-filter/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://mikezhylka.github.io/react_movies-list-filter/) and add it to the PR description.
```

Comment:
Add demo link to the PR description 
<img width="936" alt="image" src="https://github.com/user-attachments/assets/570c4021-a7b6-4007-9ec3-0504af091b7b">

Reply:
sry, thanks :)